src/index.html

```diff
@@ -16,17 +19,34 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page__body">
+    <form
+      class="search-bar search-bar-big"
+      data-qa="big"
+    >
```

Comment:
The attributes for the form tag are correctly formatted since there are more than two attributes. However, ensure that the line length does not exceed 80 characters.